### PR TITLE
Update privacy policy for Google Analytics

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**

### CODE REVIEW

File: src/privacy-policy.md.njk [hunk 1]
```diff
@@ -7,17 +7,17 @@ layout: layout-single-page-prose.njk
 
 # Privacy notice: how we use your data
 
-The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users. 
+The [GOV.UK Design System](/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users.
```
Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:10]:
```suggestion
The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users.
```

Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:10]:
```suggestion
The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users.
```

File: src/privacy-policy.md.njk [hunk 2]
```diff
@@ -7,17 +7,17 @@ layout: layout-single-page-prose.njk
...
+The GOV.UK Design System is provided by the [Government Digital Service (GDS)](https://www.gov.uk/government/organisations/government-digital-service/about) which is part of the Cabinet Office. The data controller for GDS is the Cabinet Office - a data controller determines how and why personal data can be processed.
 
 [Read the Cabinet Office’s entry in the Data Protection Public Register for more information](https://ico.org.uk/ESDWebPages/Entry/Z7414053).
 
 ## Why we need your data
 
-For a number of the activities that we undertake to complete our function, we need to process personal data. The purposes are to manage and coordinate the [GOV.UK Design System](https://design-system.service.gov.uk/) and contributions from the community. 
+For a number of the activities that we undertake to complete our function, we need to process personal data. The purposes are to manage and coordinate the [GOV.UK Design System](/) and contributions from the community.
```
Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:20]:
```suggestion
For a number of the activities that we undertake to complete our function, we need to process personal data. The purposes are to manage and coordinate the [GOV.UK Design System](https://design-system.service.gov.uk/) and contributions from the community.
```

File: src/privacy-policy.md.njk [hunk 3]
```diff
@@ -54,16 +67,17 @@ If you have signed up to our mailing list, your email address will only be retai
...
-We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s processed and when it’s stored. 
+We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s processed and when it’s stored.
 
 Your personal data may be transferred outside the United Kingdom while being processed by GDS. If this happens, we’ll make sure you’re given the same level of technical and legal protection as you are within the United Kingdom.
 
-## Who we share your data with 
+## Providers we use
 
-As part of [GOV.UK Design System](https://design-system.service.gov.uk/) we share your personal data with data processors who provide us with:
+As part of [GOV.UK Design System](/) we share your personal data with data processors who provide us with:
```
Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:76]:
```suggestion
As part of [GOV.UK Design System](https://design-system.service.gov.uk/) we share your personal data with data processors who provide us with:
```

File: src/privacy-policy.md.njk [hunk 4]
```diff
@@ -27,7 +27,7 @@ This includes:
 
 ## What data we collect from you
 
-We collect certain information and data about you when you use the [GOV.UK Design System](https://design-system.service.gov.uk/). 
+We collect certain information and data about you when you use the [GOV.UK Design System](/).
```
Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:30]:
```suggestion
We collect certain information and data about you when you use the [GOV.UK Design System](https://design-system.service.gov.uk/).
```

File: src/privacy-policy.md.njk [hunk 5]
```diff
@@ -7,17 +7,17 @@ layout: layout-single-page-prose.njk
 
 # Privacy notice: how we use your data
 
-The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users. 
+The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users.
```
Comment (calvin-lau-sig7) [inline src/privacy-policy.md.njk:10]:
```suggestion
The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users. 
```

**Review by SaraC19 — APPROVED** [review]
Looks good to me. Thank you very much!

**Review by vanitabarrett — APPROVED** [review]

**Comment by vanitabarrett** [comment]
@calvin-lau-sig7 This looks good to me! 🎉 Are you happy for this to be merged?

**Comment by calvin-lau-sig7** [comment]
> @calvin-lau-sig7 This looks good to me! 🎉 Are you happy for this to be merged?

Happy here as well! Ready for merge or 'ready to release.'

**Pull request merged**